spec/runners/unified/ddl_operations.rb

```diff
@@ -17,7 +17,14 @@ def create_collection(op)
         if session = args.use('session')
           opts[:session] = entities.get(:session, session)
         end
-        database[args.use!('collection')].create(**opts)
+        collection_opts = {}
+        if timeseries = args.use('timeseries')
+          collection_opts['timeseries'] = timeseries
+        end
+        if expire_after_seconds = args.use('expireAfterSeconds')
+          collection_opts['expireAfterSeconds'] = expire_after_seconds
```

Comment:
The `collection_opts` are the options documented in https://docs.mongodb.com/ruby-driver/master/api/Mongo/Collection.html#initialize-instance_method, which are Ruby options. If this method accepts server-named camel cased options (e.g. `writeConcern`), that should be documented. Otherwise the options here need to be converted to Ruby spelling, I suggest `:time_series` and `:expire_after`. Note that in Ruby options we give times in seconds where feasible and we do not use unit suffixes.